### ISSUE TITLE
Update Project Profile: Website Add Siyun Feng

### ISSUE THREAD

**Issue by daras-cu** [hackforla/website#7774]
### Prerequisites
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started](https://www.hackforla.org/getting-started) page and attend an onboarding session.
2. You have already read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md).

### Overview
We need to keep project information up to date so that visitors to the website can find accurate information.

### Action Items
- [x] In your IDE, open the `_projects/website.md` file.
- [x] Observe the existing syntax of the front matter block [^1] in the file.
- [x] Find the `leadership` variable and add the following profile to the end after other "Merge Team" members:
```
  - name: Siyun Feng
    github-handle: siyunfeng
    role: Merge Team
    links:
      slack: https://hackforla.slack.com/team/U07QWL2A5BN
      github: https://github.com/siyunfeng
    picture: https://avatars.githubusercontent.com/siyunfeng
```
- [x] Verify the changes by viewing the following in your local environment and include before and after screenshots with your pull request:
  - [x] Website page [^2]

### Resources/Instructions
[^1]: [Info about the front matter block](https://jekyllrb.com/docs/front-matter/)
[^2]: Project detailed info page URL: https://www.hackforla.org/projects/website)

**Comment by HackforLABot** [comment]
Hi @DDVVPP, thank you for taking up this issue! Hfla appreciates you :)

Do let fellow developers know about your:-
i. **Availability:** (When are you available to work on the issue/answer questions other programmers might have about your issue?)
ii. **ETA:** (When do you expect this issue to be completed?)

You're awesome!

P.S. - You may not take up another issue until this issue gets merged (or closed). Thanks again :)

**Comment by DDVVPP** [comment]
i. Availability:
- Feb 6: 11am - 5pm
- Feb 8: 11am - 6pm
- Feb 10: 10am - 6pm
- Feb 11-13: 12pm - 5pm

ii. ETA: 
- Feb 6 by 12:30pm

**Comment by DDVVPP** [comment]
Update:
1. Progress: My PR is waiting on 2 approved reviews
2. Blockers: There seems to be a little difficulty with a branch conflict when approving my PR from one reviewer
3. Availability: M-F: 11am - 5pm
4. ETA: Depends on when my PR gets approved
